apps/data/database/populatedb.sql

```diff
@@ -46,3 +46,24 @@ INSERT INTO user_searches (user_id, search_term) VALUES
 (3, 'textbook'),
 (4, 'FUNNY THING'),
 (5, 'Unverified User');
+
+INSERT INTO organizations (name, logo_url, donated, receiving) VALUES
+('JPetStore', 'https://api.dicebear.com/8.x/avataaars/svg?seed=OrganizationOne', 100, FALSE),
+('Costco', 'https://api.dicebear.com/8.x/avataaars/svg?seed=OrganizationTwo', 419.68, TRUE),
+('Organs-i-ate-em', 'https://api.dicebear.com/8.x/avataaars/svg?seed=OrganizationThree', 300, TRUE),
+('Moms Against Seedless Watermelons', 'https://api.dicebear.com/8.x/avataaars/svg?seed=OrganizationFour', 0.01, FALSE),
+('Seedless Watermelons Against Moms', 'https://api.dicebear.com/8.x/avataaars/svg?seed=OrganizationFive', 12345678.49, FALSE);
+
+INSERT INTO charities(name, description, start_date, end_date, image_url) VALUES
+('Charity One', 'A nice clever description for a drab default charity', '2021-01-01', '2021-12-31', 'https://api.dicebear.com/8.x/avataaars/svg?seed=CharityOne'),
+('Charity Two', 'Description for charity two', '2024-01-01', '2025-06-14', 'https://api.dicebear.com/8.x/avataaars/svg?seed=CharityTwo'),
+('Bikes4Kids', 'Buying little bicycles for poor little children', '2023-01-01', '2024-12-31', 'https://api.dicebear.com/8.x/avataaars/svg?seed=CharityThree'),
+('Little Treat', 'Fundraise so Anika can afford to have a little treat :-) <3', '2020-01-01', '2026-12-31', 'https://api.dicebear.com/8.x/avataaars/svg?seed=CharityFour'),
+('Charity Five', 'Description for charity five', '2023-01-01', '2027-12-31', 'https://api.dicebear.com/8.x/avataaars/svg?seed=CharityFive');
```

Comment:
Disappointed at the lack of creativity on the description for charities 2 and 5

Reply:
Resolved this one too